### PR TITLE
Pt br translations

### PR DESCRIPTION
**Pull request by abnersajr**
## Description
This PR bring to us the Brazilian Portuguese translation.

## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :white_circle: Improvement (Change that improves the code. Maybe performance or development improvement)
- :white_circle: Feature (Change that adds new functionality)
- :radio_button: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :radio_button: It's documentation, so the checklist items does not fit well to this scenario.

### CODE REVIEW

**Comment by ZeroX-DG** [comment]
Thanks, do you know someone who can review this PR?

File: docs/pt_BR/build.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Nós usamos o Grunt para automatizar o desenvolvimento.
+Você pode gerar o programa usando `grunt`. Contudo, nós não recomendamos isso porque a tarefa padrão inclui _codedesign_ e _authenticode_.
+
+Então nós preparamos um _script_ separado, o qual somente cria um executável.
+
+```
+grunt pre-build
+```
+
+Você irá encontrar o executável na pasta `dist`. Nota: o atualizador automático não funciona por que o app não está certificado.
```
Comment (davidsilveira) [inline docs/pt_BR/build.md]: Use `porque` instead of `por que`

File: docs/pt_BR/build.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Ambiente
+
+- npm: 6.x
+- node: 8.x
+
+## Desenvolvimento
+
+Nós usamos o Webpack HMR para desenvolver o Boostnote.
+Executando os seguintes comandos, no diretório raiz do projeto, irá iniciar o Boostnote com as configurações padrão.
```
Comment (davidsilveira) [inline docs/pt_BR/build.md]: I guess `será iniciado` is better than `irá iniciar` in this context.
You also can use `Ao executar os seguintes comandos no diretório raiz do projeto, o Boostnote será iniciado com as configurações padrão.`

**Comment by abnersajr** [comment]
All the suggestions were good. Thanks for your review @davidsilveira

**Comment by abnersajr** [comment]
@ZeroX-DG need more reviewers?

**Comment by ZeroX-DG** [comment]
@Rokt33r for PR that provide translation, how many reviews do we need before we can merge it?

**Pull request merged**